### ISSUE TITLE
[BUG] Incorrect references in `devicon.json`

### ISSUE THREAD

**Issue by lunatic-fox** [devicons/devicon#1502]
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### Bug description

In `devicon.json` some entry objects are with incorrect references in `versions.svg[]` or `aliases[]`.

***Example in heroku entry***
- [ x ] SVG array equals SVG files
~~~ json5
{
    "svg": [
        "original",
        "original wordmark",
        "plain",
        "plain-wordmark"
    ],
    "font": [
        "original",
        "original wordmark",
        "plain",
        "plain-wordmark",
        "line",
        "line-wordmark"
    ]
}
~~~

### Possible fixes or solutions

It's possible to create a program to fix the entire file or make separated PRs to fix each entry.

**Comment by Snailedlt** [comment]
I don't understand this issue. Could you rephrase?

**Comment by lunatic-fox** [comment]
Updated and re-writed in Python analyzer: [***gist***](https://gist.github.com/lunatic-fox/60e5eb541b299812b3c123fdc0cf4668)

By doing that, one idea came to my mind: maybe we can lint error messages as "yml", that way we have some highlights text, which seems to be good. 🙂

**Comment by ConX** [comment]
@lunatic-fox I was about to submit an issue for the same problem, but I had only found the `codepen` and `heroku` aliases when attempting to build the `develop` branch.

I think that it would be best to modify your script to make the necessary changes and apply them to the `develop` branch in a new PR. Additionally, your script should become part of the actions that run when a new-icon/update-icon PR is made.

If you and @Snailedlt agree, I can work during the weekend on either of those.

**Comment by Snailedlt** [comment]
@ConX Sounds like a good idea to make the changes in a new PR. 

> I think that it would be best to modify your script to make the necessary changes and apply them to the develop branch in a new PR.

Good idea! I think it would be better to do the changes in a separate script though. That way we can have one script for analyzing the errors, and another (which uses the analyzer script) to make the necessary changes.

> Additionally, your script should become part of the actions that run when a new-icon/update-icon PR is made.

Agreed, that would be this workflow I think: https://github.com/devicons/devicon/blob/1119b9f84c0290e0f0b38982099a2bd027a48bf1/.github/workflows/build_icons.yml

**Comment by ConX** [comment]
@Snailedlt I  just submitted a PR (#1623) which is based on this script by @lunatic-fox. Let me know what you think about it.
